cypress.config.js

```diff
@@ -1,8 +1,27 @@
 const { defineConfig } = require('cypress');
+const { faker } = require('@faker-js/faker');
 
 module.exports = defineConfig({
   e2e: {
     setupNodeEvents(on, config) {
-    }
-  }
+      on("task", {
+        generateUser() {
+          randomIndex = Math.floor(Math.random() * 3);
+          genders = ['Male', 'Female', 'Other'];
+          hobbies = ['Sports', 'Reading', 'Music'];
```

Comment:
```suggestion
          const randomIndex = Math.floor(Math.random() * 3);
          const genders = ['Male', 'Female', 'Other'];
          const hobbies = ['Sports', 'Reading', 'Music'];
```
I don't know, why does it work but it works)